packages/help-center/src/components/help-center-recent-conversations.tsx

```diff
@@ -80,12 +81,19 @@ const HelpCenterRecentConversations: React.FC = () => {
 			setLastConversation( lastConversation );
 			setLastMessage( lastMessage );
 
-			const { unreadConversations, unreadMessages } = getUnreadNotifications( conversations );
+			const { unreadConversations, unreadMessages } = getUnreadNotifications(
+				conversationsWithUpdatedStatuses
+			);
 			setUnreadConversationsCount( unreadConversations );
 			setUnreadMessagesCount( unreadMessages );
-			setConversations( conversations );
+			setConversations( conversationsWithUpdatedStatuses );
 		}
-	}, [ isChatLoaded, supportInteractionsResolved, supportInteractionsOpen ] );
+	}, [
+		isChatLoaded,
+		supportInteractionsResolved,
+		supportInteractionsOpen,
+		getUnreadNotifications,
```

Comment:
How come `getUnreadNotifications` wasn't a dependency before 🤔

Reply:
It was missed when the useEffect was last modified. We need to look for this throughout the package to ensure the dependency arrays are appropriately populated.